diesel_codegen_shared/src/schema_inference/pg.rs

```diff
@@ -74,19 +88,51 @@ mod information_schema {
     }
 }
 
-pub fn determine_column_type(attr: &ColumnInformation) -> Result<ColumnType, Box<Error>> {
+pub fn camel_cased(snake_case: &str) -> String {
```

Comment:
OK on changing the method name.

Right now, this PR converts a Postgres type name or variant `foo_bar` into `FooBar` on the Rust side. There are bool arguments to the macro expansion methods to control whether this happens, but they aren't currently used because there is no macro-level interface for toggling them.

I think it makes sense to map to Rust-style naming, but I am willing to be convinced otherwise. If any name manging could happen, clearly documenting default behavior and providing an option to turn it off are also necessary.